meet-data/napf/1103/meet.csv

```diff
@@ -0,0 +1,2 @@
+Federation,Date,MeetCountry,MeetState,MeetTown,MeetName
+USAPL,2011-03-05,USA,OH,Columbus,GNC Pro DL
```

Comment:
USAPL shouldn't be in the NAPF folder, but it's not clear what sanction it was under.